src/main/java/uk/gov/hmcts/reform/unspec/service/CoreCaseDataService.java

```diff
@@ -47,7 +54,7 @@ public void triggerEvent(Long caseId, CaseEvent eventName) {
             CASE_TYPE,
             caseId.toString(),
             true,
-            caseDataContentFromStartEventResponse(startEventResponse)
+            caseDataContentFromStartEventResponse(startEventResponse, data)
```

Comment:
what is data here? Data is available in start event response right?

Reply:
changed data to `contentModified`